SwiftVoiceQuickstart/ViewController.swift

```diff
@@ -386,10 +384,10 @@ extension ViewController: NotificationDelegate {
     func cancelledCallInviteReceived(cancelledCallInvite: CancelledCallInvite, error: Error) {
         NSLog("cancelledCallInviteCanceled:error:, error: \(error.localizedDescription)")
         
-        /*
-         * The twilio registration binding is refreshed for 365 days upon regirstrtion and when a new
-         * push notification is dispatch.
-         * Reset the binding date in UserDefaults.
+        /**
```

Comment:
To keep things simple and given that call and cancel come in a pair I think we can just save the time for the call push.